src/Microsoft.Health.Dicom.SqlServer/Registration/DicomServerBuilderSqlServerRegistrationExtensions.cs

```diff
@@ -53,12 +53,13 @@ public static class DicomServerBuilderSqlServerRegistrationExtensions
             services.Add<SqlIndexDataStoreV1>()
                 .Scoped()
                 .AsImplementedInterfaces();
+
             services.Add<SqlIndexDataStoreV2>()
                 .Scoped()
                 .AsImplementedInterfaces();
 
             services.Add<SqlIndexDataStoreFactory>()
-                .Scoped()
+                .Transient()
```

Comment:
reason to change this to transient?

Reply:
Sorry Will and I were testing something.  Changed back. thanks for the catch!